### PR TITLE
Pickle model using sunshine example ensemble

### PR DESCRIPTION
**Pull request by pschork**
Very rough

### CODE REVIEW

File: example_model_sunshine.py [hunk 1]
```diff
@@ -109,22 +109,22 @@
...
-#     "colsample_bytree": 0.1,
-# }
+params_name = f"lg_lgbm"
+params = {
+    "n_estimators": 2000,
+    "learning_rate": 0.1,
+    "max_depth": 5,
+    "num_leaves": 2**5,
+    "colsample_bytree": 0.1,
+}
```
Comment (pschork) [inline example_model_sunshine.py:127]: WIP These params are intentionally low to get things working

File: export_pickle.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+def load_model(name):
+    path = Path(f"{MODEL_FOLDER}/{name}.pkl")
+    if path.is_file():
+        model = pd.read_pickle(f"{MODEL_FOLDER}/{name}.pkl")
+    else:
+        model = False
+    return model
+
+def neutralize(
+    df, columns, neutralizers=None, proportion=1.0, normalize=True, era_col="era", verbose=False
```
Comment (pschork) [inline export_pickle.py:28]: I had to flatten `utils.py` in order for the pickle to be unserializable with `numerai_predict` in our execution enclave

File: example_model_sunshine.py (hunk 1 above)
Comment (michael-phillips-data) [reply example_model_sunshine.py:127]: use 0.01 for 2000 trees

0.001 for 20k+

File: export_pickle.py [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+    live_data["prediction"] = live_data[model_to_submit].rank(pct=True)
+    return live_data["prediction"]
+
+# test predict method
+predictions = predict(live_data)
+predictions.to_csv(f"live_predictions_{current_round}.csv")
+
+# export pickle
+p = cloudpickle.dumps(predict)
+model_pkl = f"sunshine_{'_'.join(sys.version.split('.')[:2])}.pkl"
```
Comment (michael-phillips-data) [inline export_pickle.py:140]: maybe some comment about this naming scheme?

**Pull request closed without merging**